assets/scss/badges.scss

```diff
@@ -13,7 +13,7 @@
   background-color: $blue;
   font-size: $font-size-small;
   font-weight: 400;
-  transition: .3s ease-in-out;
+  transition: 0.3s ease-in-out;
```

Comment:
can we use the $standard-transition variable here?

Reply:
Yes, good idea!